### PR TITLE
Add specification of reviewing process

### PR DESCRIPTION
**Pull request by kasperisager**
This PR adds a proposed specification of the process to be followed when carrying out code reviews.

### CODE REVIEW

**Comment by MGApcDev** [comment]
Looks good!

File: README.md [hunk 1]
```diff
@@ -2,6 +2,25 @@
...
+
+-   __The code is well-documented__  
+    Check that publically exposed APIs are fully documented as these will be used by other developers who are likely unfamiliar with them. Internal APIs should be sufficiently documented as well.
+
+-   __The code is consistent__  
+    Check that the code fits in with the rest of the codebase and follows the code style conventions laid out by the team. Just like in collaborative writing it shouldn't be possible to tell which developer implemented which code by simply looking at it.
+
+-   __The code minimises duplication__  
+    Check that the code doesn't add unnecessary duplication and instead reuses code where possible and appropriate.
+
```
Comment (MGApcDev) [inline README.md:94]: Add section for code testing

Comment (kasperisager) [inline README.md:94]: Done!

**Comment by MGApcDev** [comment]
Approved

**Pull request merged**